include/util/settings/hexedit.h

```diff
@@ -16,6 +16,9 @@
  */
 #pragma once
 
+#include <ui/hexedit.h>
+#include <QString>
```

Comment:
Separate this includes with a blank line (see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
Done.